Code/Editor/EditorViewportWidget.cpp

```diff
@@ -947,14 +947,21 @@ void EditorViewportWidget::SetViewportId(int id)
     );
     m_editorViewportSettingsCallbacks->SetAngleSnappingChangedEvent(m_angleSnappingHandler);
 
-    m_nearFarPlaneDistanceHandler = SandboxEditor::NearFarPlaneChangedEvent::Handler(
-        [this](float /*nearFarPlaneDistance*/)
+    m_nearPlaneDistanceHandler = SandboxEditor::NearFarPlaneChangedEvent::Handler(
+        [this](float /*nearPlaneDistance*/)
         {
             OnDefaultCameraNearFarChanged();
         }
     );
-    m_editorViewportSettingsCallbacks->SetFarPlaneDistanceChangedEvent(m_nearFarPlaneDistanceHandler);
-    m_editorViewportSettingsCallbacks->SetNearPlaneDistanceChangedEvent(m_nearFarPlaneDistanceHandler);
+    m_editorViewportSettingsCallbacks->SetNearPlaneDistanceChangedEvent(m_nearPlaneDistanceHandler);
+
+    m_farPlaneDistanceHandler = SandboxEditor::NearFarPlaneChangedEvent::Handler(
+        [this](float /*farPlaneDistance*/)
```

Comment:
uber nit: While we're here would you be able to update this to 

```
[[maybe_unused]] float farPlaneDistance
...
[[maybe_unused]] float nearPlaneDistance
```

Just so it's easier to debug in future to you can see what the value of near/farPlaneDistance

Reply:
Here you are, sorry for the late update.

Reply:
Amazing thank you! 🙂